src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Store/ContactDiaryStore.swift

```diff
@@ -875,6 +875,10 @@ class ContactDiaryStore: DiaryStoring, DiaryProviding, SecureSQLStore {
 		return removeAllEntries(from: "ContactPerson")
 	}
 
+	func removeAllCoronaTests() -> Result<Void, SecureSQLStoreError> {
```

Comment:
Could you add a unit test for that funktion, please.

Reply:
done